### ISSUE TITLE
How to get the camera intrinsic parameter ?

### ISSUE THREAD

**Issue by LiZhenLiangLee** [YvanYin/VNL_Monocular_Depth_Prediction#22]
Hi. 
In the process of point cloud reconstruction, you use camera intrinsic parameter, such as focal length and 2D coordinate of optical center. Is that all these parameters contain in the NYUD-v2 and KITTI dataset ? I mean that for each image in both datasets, it has its corresponding camera intrinsic parameter?

**Comment by YvanYin** [comment]
Hi, NYUD and KITTI have provided the camera parameters for point cloud reconstruction.

**Comment by LiZhenLiangLee** [comment]
> Hi, NYUD and KITTI have provided the camera parameters for point cloud reconstruction.

Thanks for reply

**Comment by sky-whs** [comment]
@YvanYin 
Thank you for the sophisticated work.  When I used your network for training (only the loss function WCE),  I could not have a wonderful result as your paper showed ( delta1 : 81.1). I only got (delta1: 0.40). So, may I have your training code? my mail is 464586532@qq.com. 

Best wishes!

**Comment by YvanYin** [comment]
Hi, I have released the training code. You can directly use that to train on NYU.
